Verovio.xcodeproj/project.pbxproj

```diff
@@ -2637,10 +2637,9 @@
 			};
 			buildConfigurationList = 8F086EA3188534680037FD8E /* Build configuration list for PBXProject "Verovio" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
```

Comment:
Can you discard the changes on this file?

Reply:
X-Code wanted this change, which seems understandable to me. 
You may check this: 
https://stackoverflow.com/questions/7051120/why-doesnt-my-file-move-into-en-lproj-but-instead-into-a-new-english-lproj

But if you still want it to be reverted, I'll do so.

Reply:
> https://stackoverflow.com/questions/7051120/why-doesnt-my-file-move-into-en-lproj-but-instead-into-a-new-english-lproj

This is from 8 years ago... What version of XCode are you using? I am on 10.1

Reply:
I don't know if it is identical in purpose, but there is a environmental variable called `$MACOSX_DEPLOYMENT_TARGET` that specifies the oldest MacOS operating system that the code should be compiled to work on (such as "10.4" to run on computers as old as MacOS 10.4).  If the variable is not set, then only the version of the MacOS on the compiling computer and later can be used to run the compiled program.